### ISSUE TITLE
 Screen reader does not announce paragraph information on Add card button in browse mode.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14168]
### Steps to reproduce:
1. Open above URL and login with valid credentials.
2. Navigate to 'Dashboard view' by tab key and activate through enter key.
3. Start NVDA and navigate to 'Add card' button by tab key and activate through enter key.
4. Verify that screen reader does announce paragraph information on Add card button in browse mode or not.

### Actual behavior:
'Button Domains', 'Button role-based access for admins'
Screen reader does not announce paragraph information on Add card button in browse mode. Screen reader announces name of the Add card button.

### Expected behavior:
Screen reader should announce paragraph information also on Add card button in browse mode. 

### System configuration
Browser:  Edge Version 105.0.1343.42 (official build) (64-bit) 
Version: 21H2(OS Build 22000.856)

URL: [Microsoft OneDriveOS](https://admin-ignite.microsoft.com/Adminportal/Home#/homepage) 

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2022.2.3

#### Windows version:
Win 11

#### Name and version of other software in use when reproducing the issue:
N/A

#### Other information about your system:
N/A

### Other questions
N/A
#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
N/A

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
We are unable to process this issue as it stands.

- To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).
- This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.

Kind Regards,
NV Access Software Developers

**Comment by CyrilleB79** [comment]
Also is it a duplicate of #13765?
